airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -211,6 +213,33 @@ public void writeStandardSyncOperation(final StandardSyncOperation standardSyncO
     return persistence.listConfigs(ConfigSchema.STANDARD_SYNC_OPERATION, StandardSyncOperation.class);
   }
 
+  public SourceOAuthParameter getSourceOAuthParams(final UUID SourceOAuthParameterId)
```

Comment:
We're going to need to overwrite these calls to use the secrets persistence @airbyte-jenny